include/LightGBM/config.h

```diff
@@ -970,6 +970,26 @@ struct Config {
 
   #pragma endregion
 
+  #pragma region CTR Parameters
+
+  // desc = ways to convert categorical features, currently supports:
+  // desc = ctr[:prior], where prior is a real number used to smooth the calculation of CTR values
+  // desc = count, the count of the categorical feature value in the dataset
+  // desc = for example "ctr:0.5,ctr:0.0:count will convert each categorical feature into 3 numerical features, with the 3 different ways separated by ','.
+  std::string cat_converters = "";
+
+  // desc = whether to keep the original feature values after the dataset is constructed
+  // desc = if set false, then once dataset is constructed, the cat_converters cannot be changed through parameters in train method
+  bool keep_raw_cat_data = false;
```

Comment:
keep_raw_cat_data can be removed in this version? we can refine these together with linear and other dataset parameters.